website/docs/docs/deploy/job-scheduler.md

```diff
@@ -31,7 +31,7 @@ Familiarize yourself with these useful terms to help you understand how the job
 | Over-scheduled job | A situation when a cron-scheduled job's run duration becomes longer than the frequency of the job’s schedule, resulting in a job queue that will grow faster than the scheduler can process the job’s runs. |
 | Prep time | The time dbt Cloud takes to create a short-lived environment to execute the job commands in the user's cloud data platform. Prep time varies most significantly at the top of the hour when the dbt Cloud Scheduler experiences a lot of run traffic. |
 | Run | A single, unique execution of a dbt job. |
-| Run slot | Run slots control the number of jobs that can run concurrently. Each account has a fixed number of run slots, depending on the plan tier, that are shared across projects in the account. Each running job occupies a run slot for the duration of the run, so purchasing more run slots enables more jobs to execute in parallel. |
+| Run slot | Run slots control the number of jobs that can run concurrently. Each account has a fixed number of run slots, depending on the plan tier, that are shared across projects in the account. Each running job occupies a run slot for the duration of the run, so if you need more jobs to execute in parallel consider the enterprise tier. |
```

Comment:
@schottj should we link to the [pricing page](https://www.getdbt.com/pricing/)?

Reply:
sure!

Reply:
hey @schottj , i think you opened a separate pr addressing the run slots. could we combine your changes here onto the other pr: https://github.com/dbt-labs/docs.getdbt.com/pull/3712?

Reply:
Mirna, you can change the base branch to PR #3712  or make this PR the base branch and merge them together (this might be what you were suggesting too.)